### DISCUSSION TITLE
cannot run executable found relative to current directory error on windows install

### DISCUSSION THREAD

**Discussion by flips22** [garethgeorge/backrest#266] [Q&A]
I'm a bit puzzled on what I did to get my backrest install borked, so I'm hoping someone out there can give me some direction. I've installed backrest on another PC and it has worked fine.  Tonight I was setting up on a different windows PC and I was having trouble to a rest server that I was also setting up for the first time. I was getting password errors and such, so I was getting that straighten out on the rest server docker.  

I think my issue started when I tried to run the restic-0.16.4.exe on a command line to try to init a rest server repo outside of the backrest UI.  But I'm not entirely positive on that point. I don't have a restic folder in appdata, so I'm not sure there doesn't seem to be a cache folder.  This is reflected in the restic cache command as well.

At this point whenever I try to create a repo in the UI, even just a local folder, I get the following error:

Operation failed: [unknown] failed to init repo: init failed: command "restic-0.16.4.exe init --json -o sftp.args=-oBatchMode=yes" failed: exec: "restic-0.16.4.exe": cannot run executable found relative to current directory

I have backrest in the program files folder, I'm running the command prompt as admin, I've run the uninstall.sh file, I've deleted the program files folder, and the appdata folder and reinstalled and even restarted the windows machine, and I still get the same error.

I appreciate if anyone can give me some idea on what to try next.

**Comment by garethgeorge** [comment]
Hmm. I think your install matches how I run backrest on Windows (re: where things are installed) so that’s a bit surprising—

I’d expect the restic command to be the full path to the binary e.g. including C:\ProgramFiles\… 

are you overriding BACKREST_RESTIC_COMMAND or anything of that sort?

**Reply by flips22** [reply]
Thank you Gareth.  I am not setting that or any environment variables, but you gave me an idea.

I set BACKREST_RESTIC_COMMAND = C:\Program Files\Backrest\restic-0.16.4.exe and it works perfectly now.

If you would like me to try something to try to root cause it more, just let me know.

Thanks for this great app, and all your efforts on this.  I been wanting to get a better back up solution for a while now, but I was hesitant to work solely in the command line. This is exactly what I was looking for!
